pkgs/development/python-modules/scikit-bio/default.nix

```diff
@@ -1,70 +1,75 @@
 {
   lib,
   buildPythonPackage,
-  fetchPypi,
+  fetchFromGitHub,
+  isPy3k,
+  setuptools,
   cython,
-  lockfile,
-  cachecontrol,
+  oldest-supported-numpy,
+  requests,
   decorator,
-  h5py,
-  ipython,
-  matplotlib,
   natsort,
   numpy,
   pandas,
   scipy,
+  h5py,
   hdmedians,
-  scikit-learn,
-  coverage,
+  biom-format,
   python,
-  isPy3k,
+  pytestCheckHook,
+  coverage,
 }:
 
 buildPythonPackage rec {
-  version = "0.6.0";
-  format = "setuptools";
   pname = "scikit-bio";
+  version = "0.6.0";
+  pyproject = true;
+
   disabled = !isPy3k;
 
-  src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-EBBafDwVrlkQJEkn8punqjUjSxnr5lE7hIRUc0OywQ8=";
+  src = fetchFromGitHub {
+    owner = "scikit-bio";
+    repo = "scikit-bio";
+    rev = "refs/tags/${version}";
+    hash = "sha256-v8/r52pJpMi34SekPQBf7CqRbs+ZEyPR3WO5RBB7uKg=";
   };
 
-  nativeBuildInputs = [ cython ];
-  nativeCheckInputs = [ coverage ];
-  propagatedBuildInputs = [
-    lockfile
-    cachecontrol
+  build-system = [
+    setuptools
+    cython
+    oldest-supported-numpy
+  ];
+
+  dependencies = [
+    requests
     decorator
-    ipython
-    matplotlib
     natsort
     numpy
     pandas
     scipy
     h5py
     hdmedians
-    scikit-learn
+    biom-format
   ];
 
-  # cython package not included for tests
-  doCheck = false;
+  nativeCheckInputs = [
+    pytestCheckHook
+    coverage
```

Comment:
```suggestion
```
Could you please remove `coverage` and patch if necessary?
We don't need to check the coverage.

Reply:
Removed, no issues